code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm

```diff
@@ -30,22 +30,26 @@
 	. = ..()
 	if(!iscarbon(target))
 		return
+	var/mob/living/carbon/C = target
+	var/atom/throw_target = get_edge_target_turf(C, user.dir)
+	if(!C.anchored)
+		. = TRUE
+		C.throw_at(throw_target, rand(4,8), 14, user)
+	return
```

Comment:
are we still sticking with this yeeting effect? wouldn't, say, temporary blindness or something be more appropriate? would it be possible to apply the blindness before the message from the mansus grasping is displayed, possibly concealing your identity from your victim's chat log?

Reply:
there is a pr up for that already